docs/vale/styles/config/vocabularies/Dagster/accept.txt

```diff
@@ -1,72 +1,81 @@
-[Cc]onfig
-\bDagster\b
+ACR
```

Comment:
Alphabetized and moved acronyms to separate paragraph.

I originally thought we could remove these acronyms as they are in `docs/vale/styles/Dagster/acronyms.yml` but "term" errors are thrown if not present in this file.